core/formats/columnstore2.hpp

```diff
@@ -94,14 +94,22 @@ class column final : public irs::column_output {
       return offset_[-1];
     }
 
-    void push_back(uint64_t offset) noexcept {
+    bool push_back(uint64_t offset) noexcept {
       IRS_ASSERT(offset_ < offsets_ + kBlockSize);
+      if (!(offset_ < offsets_ + kBlockSize))
+        return false;
+
       *offset_++ = offset;
+      return true;
     }
 
-    void pop_back() noexcept {
+    bool pop_back() {
```

Comment:
Why remove the `noexcept` specifier?
```suggestion
    bool pop_back() noexcept {
```

Reply:
This was done to test something and forgot to put it back.
But it's fixed now.